source/sass/patterns/molecules/reference.scss

```diff
@@ -24,7 +25,8 @@
 
 .reference__doi {
   @include covert-link() {
-    display: block;
+    display: table;
```

Comment:
Can we not use inline-block instead? It's not really a table of data as it's just one url.

Reply:
:+1:, was confused why that didn't work initially but with `inline-size: auto` it does! The `all: inherit` is becoming more trouble than it's worth.

(`display: table` doesn't provide semantics.)